src/main/kotlin/step4/CarRacing.kt

```diff
@@ -0,0 +1,24 @@
+package step4
+
+import step4.model.Race
+import step4.model.Referee
+import step4.strategy.ConditionalMoveStrategy
+import step4.view.InputView
+import step4.view.ResultView
+
+fun main() {
+    val nameListOfCars = InputView.getNameListOfCars()
+    val numberOfTrials = InputView.getNumberOfTrials()
+
+    val race = Race(nameListOfCars)
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    val race = Race(nameListOfCars, numberOfTrials)
```

Reply:
이 내용과 Race가 name을 프로퍼티로 가지지 않아도 될 것 같다고 하신 의견을 추합하였습니다.
Race의 initiate 함수를 이용하여 nameListOfCars, numberOfTrials 를 전달받도록 수정하였습니다.